docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py

```diff
@@ -35,32 +35,35 @@ def set_up_context_for_example(context):
 # <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - full code example">
 import great_expectations as gx
 
+# Define your custom SQL query.
+# <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - define query">
+my_query = """
+    SELECT
+        *
+    FROM
+        {batch}
+    WHERE
+        passenger_count > 6 or passenger_count < 0
+    """
+# </snippet>
 
-# Define a custom Expectation that uses SQL by subclassing UnexpectedRowsExpectation
-# <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - define a custom UnexpectedRowsExpectation">
-# <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - define the query for an UnexpectedRowsExpectation">
-# <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - define a more descriptive name for an UnexpectedRowsExpectation">
-class ExpectPassengerCountToBeLegal(gx.expectations.UnexpectedRowsExpectation):
-    # </snippet>
-    unexpected_rows_query: str = (
-        "SELECT * FROM {batch} WHERE passenger_count > 6 or passenger_count < 0"
-    )
-    # </snippet>
-    description: str = "There should be no more than **6** passengers."
-
+# Customize how the Expectation renders in Data Docs.
+# <snippet name="docs/docusaurus/docs/core/customize_expectations/_examples/use_sql_to_define_a_custom_expectation.py - define description">
+my_description = "There should be no more than **6** passengers."
```

Comment:
Please don't take action on this now, but just throwing it out as a stylistic thing in case other folks feel similarly - I personally find it much more to take in when we spread out simple stuff across multiple snippets, i.e., I'd rather just see this inlined when creating the expectation. But again, that's just my 2 cents!